addons/mrp/views/mrp_workcenter_views.xml

```diff
@@ -299,20 +299,24 @@
                     <sheet>
                         <div class="oe_button_box" name="button_box">
                             <field name="has_routing_lines" invisible="1"/>
-                            <button string="Operations" type="object"
+                            <button type="object"
                                 name="action_show_operations"
                                 attrs="{'invisible': [('has_routing_lines', '=', False)]}"
                                 context="{'default_workcenter_id': active_id}"
-                                class="oe_stat_button" icon="fa-cog"/>
+                                class="oe_stat_button" icon="fa-cog">
+                                <div class="o_stat_info">
+                                    <span class="o_stat_text">Operations</span>
+                                </div>
+                            </button>
```

Comment:
seems like you need to do this same change in stock/views/product_views for these 3 buttons:

https://github.com/odoo/odoo/blob/saas-16.3/addons/stock/views/product_views.xml#L322-L336

I didn't check any if any other buttons need updating, I only noticed them when looking at the result of your changes.

Reply:
@ticodoo Good catch! Thanks 👍